### PR TITLE
Feature/date pattern and apidocs

### PR DESCRIPTION
**Pull request by CarlosFeldmann**
- creating serializer/deserializer for LocalDate for object mapper
- more examples with OpenAPI specification
- documentation config

### CODE REVIEW

File: src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/config/exceptions/dtos/ResourceNotFoundError.java [hunk 1]
```diff
@@ -2,12 +2,15 @@
 
 
 import br.com.mercadolivre.bootcampw2.grupo11.socialmeli.exceptions.ResourceNotFoundException;
+import io.swagger.v3.oas.annotations.media.Schema;
 import lombok.Getter;
 
 @Getter
 public class ResourceNotFoundError extends ApiError {
 
+    @Schema(description = "Resource type that was't present on the server", example = "User")
```
Comment (Victor-Planas) [inline src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/config/exceptions/dtos/ResourceNotFoundError.java]: wasn't*

File: src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/forms/CreatePromocionalPostForm.java [hunk 2]
```diff
@@ -14,10 +15,12 @@
 
 
     @NotNull
+    @Schema(example = "true", description = "If this post have a promotional price")
```
Comment (Victor-Planas) [inline src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/forms/CreatePromocionalPostForm.java]: has*

**Review by Victor-Planas — APPROVED** [review]
Awesome, amazing, gorgeous! 😭😭

**Review by Giovannapls — APPROVED** [review]
oki doki

**Review by lmarizmeli — APPROVED** [review]
🚐

**Pull request merged**